### ISSUE TITLE
Drawer Component  css  style  issue  ?

### ISSUE THREAD

**Issue by itjayaprakash** [material-components/material-components-web-react#557]
I  tried to import drawer component but is not  reflecting css in  React App
https://github.com/material-components/material-components-web-react/tree/master/packages/drawer

app.scss file  i added  css  code it's reflecting..
```

.drawer-container {
  display: flex;
  flex-direction: row;
  height: 100vh;
  overflow: hidden;
}

.drawer-app-content {
  flex: auto;
  overflow: auto;
}
```

but this style css not working.

import '@material/react-drawer/index.scss';
import "@material/react-drawer/dist/drawer.css";

**Step 1**
i have created React App. using given below cmd
```
npx create-react-app my-app
cd my-app
npm start

```
**Step 2** 
Git command  propmt

`SET SASS_PATH=.\node_modules  `

after that 

`npm install node-sass`

**Step 3**
git command  propmt

```
npm install @material/react-drawer
npm install @material/react-list
npm install @material/react-material-icon
npm install @material/react-top-app-bar
```

**Step 4**
App.js file 

```
import React, {Component} from 'react';
import TopAppBar, {TopAppBarFixedAdjust} from '@material/react-top-app-bar';
import Drawer, {DrawerAppContent, DrawerContent, DrawerHeader, DrawerTitle} from '@material/react-drawer';
import MaterialIcon from '@material/react-material-icon';
import List, {ListItem, ListItemGraphic, ListItemText} from '@material/react-list';

// includes imports for drawer, list, material icon and top app bar styles
import './App.scss';
import '@material/react-drawer/index.scss';
import "@material/react-drawer/dist/drawer.css";

export default class App extends Component {
  state = {selectedIndex: 0};

  render() {
    return (
      
      <div className='drawer-container'>
      <TopAppBar title='Inbox'/>

      <TopAppBarFixedAdjust>
        <Drawer>
          <DrawerHeader>
            <DrawerTitle tag='h2'>
              jane.smith@gmail.com
            </DrawerTitle>
          </DrawerHeader>

          <DrawerContent>
            <List singleSelection selectedIndex={this.state.selectedIndex}>
              <ListItem>
                <ListItemGraphic graphic={<MaterialIcon icon='folder'/>} />
                <ListItemText primaryText='Mail' />
              </ListItem>
            </List>
          </DrawerContent>
        </Drawer>

        <DrawerAppContent className='drawer-app-content'>
          Your inbox content
        </DrawerAppContent>
      </TopAppBarFixedAdjust>
    </div>

    );
  }
}

```

Output is like.

<img src="https://i.imgur.com/lHPtwyx.png">

**Comment by moog16** [comment]
A few things:

1. You don't need to import both:
```import '@material/react-drawer/index.scss';
import "@material/react-drawer/dist/drawer.css";```
You only need the scss file in your case.

2. You must also import the top-app-bar, list, and material-icon scss files
```
import '@material/react-top-app-bar/index.scss'
import '@material/react-list/index.scss'
import '@material/react-material-icon/index.scss'
```

3. You are missing the material-icon import in your public/index.html
```
<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">```

**Comment by ZeJ0hn** [comment]
Hello @moog16,

Thank you for your example.
I need to add this in the css file :
```
.top-app-bar-fix-adjust {
    display: flex;
    **width: 100%;**
}
```

Else my main component didn't fill the whole space.
Is it ok for you ?

Note : the documentation for the drawer is not up-to-date. The _.top-app-bar-fix-adjust_ is not defined and the main component doesn't render correctly

**Comment by moog16** [comment]
@ZeJ0hn are you saying the example on the drawer readme is not correct? 

Also I'm not sure what you mean about the `NOTE`. The class is actually `.top-app-bar--fixed-adjust`

**Comment by ZeJ0hn** [comment]
@moog16 My explanation is not very clear. Sorry for that.

In the drawer README, if you try to use the example "_Dismissible or Permanent drawer below top app bar_", it doesn't work. The text "Your inbox content" doesn't appear.
So, instead to have this css :
```
.drawer-container {
  display: flex;
  flex-direction: row;
  height: 100vh;
  overflow: hidden;
}

.drawer-app-content {
  flex: auto;
  overflow: auto;
}
```

I had to use this one.
```.drawer-container {
    display: flex;
    flex-direction: row;
    height: 100vh;
    overflow: hidden;
}

.drawer-app-content {
    flex: auto;
    overflow: auto;
}

.top-app-bar-fix-adjust {
    display: flex;
    width: 100%;
}
```

and add the class .top-app-bar-fix-adjust to the <TopAppBarFixedAdjust>

**Comment by moog16** [comment]
ah I see...I remember having to do that as well. Do you mind opening a PR to update the readme?

**Comment by ZeJ0hn** [comment]
No PR. I can do that if you want

**Comment by moog16** [comment]
Ya that would be great! Otherwise I might reopen this issue to fix them later.

**Comment by ZeJ0hn** [comment]
Hello,

I create the PR #666 
I also fix another example with the PR #665 (but I don't know with the build failed)

Have fun

**Comment by moog16** [comment]
thank you!
